hw/usb/tinyusb/README.md

```diff
@@ -0,0 +1,48 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+-->
+
+To use tinyusb add following lines to **project.yml**.
+
+```yaml
+repository.tinyusb:
+    type: github
+    vers: 0.0.0
+    user: hathach
+    repo: tinyusb
+```
+
+This dependency will start USB stack task.
+```yaml
+pkg.deps:
+    - "@apache-mynewt-core/hw/usb/tinyusb"
+```
+
+To easy add support for CDC or HID class device simply add
+**hw/usb/tinyusb/std_descriptors** package to project
+```yaml
+pkg.deps:
+    - "@apache-mynewt-core/hw/usb/tinyusb/std_descriptors"
+```
+
+To use console over USB, set syscfg value
+```yaml
+    CONSOLE_USB: 1
```

Comment:
We need also USBD_CDC to be set as it is not done automatically.
Need to figure out how to do it automatically.